### ISSUE TITLE
Validation failed on array of ContractAddress

### ISSUE THREAD

**Issue by tabaktoni** [0xs34n/starknet.js#701]
**Describe the bug**
`Validate Unhandled: argument oracles, type core::array::Array::<core::starknet::contract_address::ContractAddress>, value 0x7447084f620ba316a42c72ca5b8eefb3fe9a05ca5fe6430c65a69ecc4349b3b`

```ts
await contract.method1(
        ["0x7447084f620ba316a42c72ca5b8eefb3fe9a05ca5fe6430c65a69ecc4349b3b"],
        require
    );
```

abi
```json
{
            "type": "function",
            "name": "add_oracles",
            "inputs": [
                {
                    "name": "oracles",
                    "type": "core::array::Array::<core::starknet::contract_address::ContractAddress>"
                },
                {
                    "name": "required",
                    "type": "core::array::Array::<core::bool>"
                }
            ],
            "outputs": [],
            "state_mutability": "external"
        },
```

**Comment by PhilippeR26** [comment]
I have identified some other "exotic" Cairo V2 types that are not handled properly by Starknet.js .
I am currently analyzing them.

**Comment by PhilippeR26** [comment]
This case  (ContractAddress) is included in issue #707 .

**Comment by tabaktoni** [comment]
Resolved in https://github.com/0xs34n/starknet.js/pull/712
v5.19.0
